aep/0142.yaml

```
@@ -0,0 +1,37 @@
+functionsDir: ../functions
+functions:
+  - aep-142-time-field-type
+
+rules:
+  aep-142-time-field-type:
+    description: 'Fields with time-related suffixes should use appropriate types and formats.'
+    message: '{{error}}'
+    severity: warn
+    formats: ['oas2', 'oas3']
+    given: '$..[?(@property.match(/_(time|times|date|seconds|millis|micros|nanos)$/))]'
```

I think this pattern could potentially generate some nasty false positives. This will help cut down on that:
```suggestion
    given: '$..properties[?(@property.match(/_(time|times|date|seconds|millis|micros|nanos)$/))]'
```